src/Oscoin/Storage/Block/SQLite.hs

```diff
@@ -65,36 +60,3 @@ withBlockStore path genesisBlock action =
     in bracket (newBlockStore <$> (initialize genesisBlock =<< open path))
                (close . snd)
                (action . fst)
-
-
-lookupBlock
```

Comment:
These functions have been moved into the `SQLite.Internal` module, so that they can be reused.